composer.json

```diff
@@ -38,6 +38,7 @@
     "require-dev": {
         "filp/whoops": "^2.1",
         "phpunit/phpunit": "^7.1.1",
+        "swoole/ide-helper": "dev-master",
```

Comment:
What does this provide, exactly? Is it just stubs for autocompletion?

If so, may I request that you open a separate PR for just this change, please?